### ISSUE TITLE
VM disappears just as docker-ce is about to be unpacked for ciao singlevm setup

### ISSUE THREAD

**Issue by jdandrea** [intel/ccloudvm#95]
Trying this on a new server with no proxy requirements. (Also, `virt-host-validate qemu` passes.)

Here's what disk and memory look like just before the VM disappears:

```
jdandrea@singlevm:~$ df -k
Filesystem     1K-blocks     Used Available Use% Mounted on
udev             1966896        0   1966896   0% /dev
tmpfs             394804     5520    389284   2% /run
/dev/vda1       16197524  1167524  15013616   8% /
tmpfs            1974004        0   1974004   0% /dev/shm
tmpfs               5120        0      5120   0% /run/lock
tmpfs            1974004        0   1974004   0% /sys/fs/cgroup
tmpfs             394804        0    394804   0% /run/user/1005
hostgo          19092180 18092348      6964 100% /home/jdandrea/go

jdandrea@singlevm:~$ free -m
              total        used        free      shared  buff/cache   available
Mem:           3855         111        3238           5         505        3511
Swap:             0           0           0
```

In `/var/log/cloud-init-output.log`, it ends here:

```
Preparing to unpack .../docker-ce_18.03.1~ce-0~ubuntu_amd64.deb ...
Unpacking docker-ce (18.03.1~ce-0~ubuntu) ...
```

... and then the VM is gone (`ccloudvm instances` shows nothing).

In `syslog` on the host:

```
May 12 12:43:03 agave011 ccvm[16005]: Create {Name:singlevm WorkloadName:ciao Debug:true Update:false CustomSpec:{MemMiB:0 DiskGiB:0 CPUs:0 PortMappings:[8080-80] Mounts:[] Drives:[] Qemuport:0 HostIP:<nil>} HTTPProxy: HTTPSProxy: NoProxy: GoPath:/home/jdandrea/go} called
May 12 12:43:03 agave011 ccvm[16005]: Transaction ID 1
May 12 12:43:03 agave011 ccvm[16005]: CreateResult(1) called
May 12 12:43:03 agave011 ccvm[16005]: Downloading https://cloud-images.ubuntu.com/xenial/current/xenial-server-cloudimg-amd64-disk1.img
May 12 12:43:03 agave011 ccvm[16005]: request sent https://cloud-images.ubuntu.com/xenial/current/xenial-server-cloudimg-amd64-disk1.img
May 12 12:43:03 agave011 ccvm[16005]: Download of xenial-server-cloudimg-amd64-disk1.img finished
May 12 12:43:03 agave011 ccvm[16005]: CreateResult(1) called
May 12 12:43:03 agave011 kernel: [ 2181.063326] kvm: zapping shadow pages for mmio generation wraparound
May 12 12:43:03 agave011 kernel: [ 2181.068503] kvm: zapping shadow pages for mmio generation wraparound
May 12 12:43:04 agave011 kernel: [ 2182.080505] kvm [16376]: vcpu0 unhandled rdmsr: 0x1c9
May 12 12:43:04 agave011 kernel: [ 2182.086512] kvm [16376]: vcpu0 unhandled rdmsr: 0x1a6
May 12 12:43:04 agave011 kernel: [ 2182.091411] kvm [16376]: vcpu0 unhandled rdmsr: 0x3f6
May 12 12:43:07 agave011 kernel: [ 2184.779224] kvm [16376]: vcpu0 unhandled rdmsr: 0x34
May 12 12:43:03 agave011 ccvm[16005]: message repeated 4 times: [ CreateResult(1) called]
May 12 12:43:09 agave011 ntpd[1579]: Soliciting pool server 104.131.53.252
May 12 12:43:11 agave011 ntpd[1579]: Soliciting pool server 216.6.2.70
May 12 12:43:20 agave011 ntpd[1579]: Soliciting pool server 91.189.89.199
May 12 12:43:22 agave011 ntpd[1579]: Soliciting pool server 66.228.42.59
May 12 12:43:24 agave011 ntpd[1579]: Soliciting pool server 2001:19f0:200:144b::1000
May 12 12:43:32 agave011 ccvm[16005]: CreateResult(1) called
```

**Comment by jdandrea** [comment]
Possible reason: If your ccloudvm host's root disk is almost full, "THERE's yer problem!" :)

**Comment by markdryan** [comment]
What does the terminal where you executed ccloudvm create show?  Are there any errors?  An instance does not show up when you do a ccloudvm instances until it has been successfully created.  If the creation fails or is cancelled (CTRL+C) the instance is automatically deleted.

**Comment by jdandrea** [comment]
On the host it just stops short, no errors, and I can ^C out of it. The VM appears to stop short, but it is in fact obliterated. I'm not sure if ccloudvm can detect that the VM disappeared though, at least at that point in the workflow.

This definitely had something to do with the disk filling up on this other host. As I freed up space, I got further along. I solved it by moving to another host with more space! :)
